.gitignore

```diff
@@ -3,3 +3,4 @@
 /interface/static/
 /.vagrant
 *.pyc
+/.vscode/
```

Comment:
The .gitignore of the repo is still not a good place. If someone for whatever reason uses intellij, they will have a .idea folder that must be ignored. This will pollute the `.gitignore` file again. The [resource](https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) I send you previously stated that you can run the following command:
```
$ git config --global core.excludesfile ~/.gitignore_global
``` 
Now any ignore rules that you add to the `.gitginore_global` will apply to all repos.